frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -67,6 +66,27 @@ end
 
 ReaderBookmark.onPhysicalKeyboardConnected = ReaderBookmark.registerKeyEvents
 
+function ReaderBookmark:genItemTextMenuItem(type, get_string)
+    local text_type = {
+        text = _("highlighted text"),
+        all  = _("highlighted text and note"),
+        note = _("note, or highlighted text"),
+    }
+    if get_string then
+        return text_type[type]
+    end
+    return {
+        text = text_type[type],
+        checked_func = function()
+            return self.items_text == type
+        end,
+        callback = function()
+            self.items_text = type
+            G_reader_settings:saveSetting("bookmarks_items_text", type)
```

Comment:
"bookmarks_items_text_type" ? (its value is not a some random text, but some type from some enumeration)